package.json

```diff
@@ -34,5 +34,22 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
- [ ] Please you don't have `react-router-dom` install a module since it is not found in your package.json file but you are using your app due to that I'm getting this error when running your project :point_down: so make sure you have react-router-dom present in your package.json
  
  ![image](https://github.com/PedroH-Neves/book-store-react/assets/73854999/02e6e616-0b21-44df-bd11-9608374fe722)

Reply:
well resolved! ✔️